app/src/main/java/com/teamltt/carcare/activity/DynamicActivity.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017, Team LTT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teamltt.carcare.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.teamltt.carcare.R;
+import com.teamltt.carcare.fragment.DatePickerFragment;
+
+public class DynamicActivity extends BaseActivity {
+    private boolean from;
+
+    public void onCreate(Bundle savedInstanceState) {
+        activityContent = R.layout.activity_dynamic;
+        includeDrawer = false;
+        super.onCreate(savedInstanceState);
+        from = true;
+    }
+
+
+    public void showDatePickerDialog(View v) {
+        from = findViewById(R.id.buttonTo) != v;
+        DatePickerFragment dialog = new DatePickerFragment();
+        dialog.show(getFragmentManager(), "DateFragment");
+    }
+
+    public void setDate(int year, int month, int day) {
+        TextView tv;
+        if (from) {
+            tv = (TextView) findViewById(R.id.textFrom);
+        } else {
+            tv = (TextView) findViewById(R.id.textTo);
+        }
+        String date = month + "/" + day + "/" + year;
```

Comment:
This should actually be changed to be much more like the wireframe before. The way we receive the date may change when we do that, so we shouldn't worry about how it's displayed